pkgs/unified_analytics/lib/src/enums.dart

```diff
@@ -153,6 +153,16 @@ enum DashEvent {
     required this.description,
     this.toolOwner,
   });
+
+  /// This takes in the string label for a given [DashEvent] and returns the
+  /// enum for that string label.
+  static DashEvent? getDashEventByLabel(String label) {
+    for (final event in DashEvent.values) {
+      if (event.label == label) return event;
+    }
+
+    return null;
```

Comment:
this can be simplified:
`return DashEvent.values.firstWhereOrNull((e) => e.label == label);`